docs/Migration-Guide.md

```diff
@@ -9,9 +9,10 @@
   - [2. Payone portal changes](#2-payone-portal-changes)
   - [3. Changes in the shops which uses the service](#3-changes-in-the-shops-which-uses-the-service)
   - [4. Setup a new tenant (branch,shop,merchant)](#4-setup-a-new-tenant-branchshopmerchant)
-  - [5. Importan notes](#5-importan-notes)
+  - [5. Important notes](#5-important-notes)
```

Comment:
Don't we consider it as a major version of the library?

Reply:
Yes sir, I just don't know which version it will be. Do you know?

Reply:
Last release is https://github.com/commercetools/commercetools-payone-integration/releases/tag/v2.11.0. So, our change will be released as v2.12.0

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/313/commits/c18460e958428b311f59d1e1605dc8280e59f21c